data/siftery.json

```diff
@@ -0,0 +1,23 @@
+{
+  "id": "siftery",
+  "name": "Siftery",
+  "description": "Share products your company uses and get personalized recommendations.",
+  "categories": [],
+  "iconUrl": "https://pbs.twimg.com/profile_images/774378268950605824/9q92Ct1P_400x400.jpg",
+  "website": "https://www.siftery.com",
+  "countryHQ": "US",
+  "gdprReadyStatus": "nonCompliant",
```

Comment:
I'm currently awaiting a response from Siftery about their compliance status.

Reply:
I wouldn't wait much more than a week for a reply. They should know whether they are working towards it, or avoiding it. If they are GDPR compliant, they would likely announce it/update privacy policy :)

Reply:
I would mark this as in progress https://track.siftery.com/gdpr